lib/utils/dotsnyk/index.ts

```diff
@@ -0,0 +1,46 @@
+import * as fs from 'fs';
+import { parse } from 'yaml';
+import { resolve } from 'path';
+
+// # Snyk (https://snyk.io) policy file
+// version: v1.14.0
+//
+// exclude:
+//   unmanaged:
+//     - test.spec.ts # single file
+//     - src/lib # single directory
+//     - tests/*.ts # any file with the “.ts” extension under “tests”
+//     - “**/*.spec.ts” # files ending with “.spec.ts” from any directory
+//     - tests?/* # files directly inside “test” and/or “tests” directories
+//     - tests/** # all files and directories inside “tests”
+//
+
+export type Config = { exclude: { unmanaged: string[] } };
+
+export function parseDotSnyk(filePath: string): Config {
+  try {
+    const dotSnykFile = fs.readFileSync(filePath, 'utf-8');
```

Comment:
maybe rename dotSnykFile to content, bytes or similar. It's pretty obvious what it contains, due to the name of the function (`parseDotSnyk`).

Reply:
fileContents?

Reply:
sounds good, maybe scratch the plural version and just call it fileContent?